setup.py

```diff
@@ -11,14 +11,33 @@
     "tensorflow>=2.4.0",
 ]
 
+with open("README.md", "r") as file:
+    long_description = file.read()
+
+with open("VERSION", "r") as file:
+    version = file.read()
```

Comment:
```suggestion
    version = file.read().strip()
```
so we don't have to worry about no-newline ?